crates/vm/levm/src/utils.rs

```diff
@@ -738,3 +738,13 @@ pub fn eip7702_get_code(
 
     Ok((true, access_cost, auth_address, authorized_bytecode))
 }
+
+pub fn build_access_list(substate: &mut Substate) -> AccessList {
```

Comment:
```suggestion
pub fn build_access_list(substate: &Substate) -> AccessList {
```
It doesn't need to be mutable, right?

Reply:
Addressed in [000f5f7](https://github.com/lambdaclass/ethrex/pull/2244/commits/000f5f772a83ffc7a245bf9a3b04936f83ca09a0)!